core/frontend/src/ViewContext.ts

```diff
@@ -325,8 +324,74 @@ export class DecorateContext extends RenderContext {
     return (undefined !== thisRay.intersectionWithPlane(dotX > dotY ? planeX : planeY, thisPt)) ? lastPt.distance(thisPt) : 0.0;
   }
 
+  public drawStandardGridNew(gridOrigin: Point3d, rMatrix: Matrix3d, spacing: XAndY, gridsPerRef: number, _isoGrid: boolean = false, _fixedRepetitions?: Point2d): void {

```

Comment:
Is this intentionally `@public`? The old ``drawStandardGrid` is marked `@internal`. Can we give it a more meaningful name than "new"?

Reply:
This is just temporary for me to look at Paul, this code isn't finished yet and is intended to replacement drawStandardGrid.